i18n/ru/package.i18n.json

```diff
@@ -0,0 +1,76 @@
+  "param.coveredDarkTheme": "Цвет фона для охватываемых линий в темной теме для покрытия ESP-IDF.",
+  "param.partialLightTheme": "Цвет фона для частично покрытых линий в светлой теме для покрытия ESP-IDF.",
+  "param.partialDarkTheme": "Цвет фона для частично покрытых линий в темной теме для покрытия ESP-IDF.",
+  "param.uncoveredLightTheme": "Цвет фона для непокрытых линий в светлой теме для покрытия ESP-IDF.",
+  "param.uncoveredDarkTheme": "Цвет фона для непокрытых линий в темной теме для покрытия ESP-IDF.",
+  "view.components.name": "Компоненты проекта",
+  "configuration.title": "ESP-IDF",
+  "espIdf.apptrace.archive.refresh.title": "Обновить список архива трассировки",
+  "espIdf.apptrace.archive.showReport.title": "Показать отчет о трассировке",
+  "espIdf.apptrace.customize.title": "Настроить параметры для трассировки приложения",
+  "trace.poll_period.description": "poll_period будет установлен для трассировки приложения",
+  "trace.trace_size.description": "trace_size будет установлен для трассировки приложения",
+  "trace.stop_tmo.description": "stop_tmoбудет установлен для трассировки приложения",
+  "trace.wait4halt.description": "wait4halt будет установлен для трассировки приложения",
+  "trace.skip_size.description": "skip_size будет установлен для трассировки приложения",
+  "param.saveBeforeBuildDescription": "Сохраните все отредактированные файлы в рабочей области, прежде чем продолжить сборку, хотя, если не удается сохранить файлы, она все равно будет построена",
+  "param.saveScope": "Куда сохранять конфигурацию с помощью команд ESP-IDF с числовым значением vscode.ConfigurationTarget. Глобально = 1, Рабочая область = 2, Папка рабочей области = 3",
+  "param.rainmaker.api.server_url": "URL-адрес облачного сервера ESP-Rainmaker",
+  "esp.rainmaker.backend.sync.title": "Синхронизация с облачным сервером ESP-Rainmaker",
+  "esp.rainmaker.backend.connect.title": "Подключится к облачному серверу ESP-Rainmaker",
+  "esp.rainmaker.backend.logout.title": "Отключить аккаунт Rainmaker",
+  "esp.rainmaker.backend.add_node.title": "Добавить новый узел",
+  "esp.rainmaker.backend.remove_node.title": "Удалить этот узел",
+  "esp.rainmaker.backend.update_node_param.title": "Обновить параметры для устройства",
+  "param.esp.rainmaker.oauth.url": "URL-адрес сервера аутентификации ESP-Rainmaker",
+  "idf.wssPort.description": "Порт сервера веб-сокетов для дампа ядра или заглушки GDB",
+  "espIdf.launchWSServerAndMonitor.title": "Запустите IDF мониторинг для режима CoreDump / GDB-Stub",
+  "esp_idf.initGdbCommands.description": "Одна или несколько команд xtensa-esp32-elf-gdb для выполнения, чтобы настроить базовый отладчик. Пример: \"initGdbCommands\": [ \"target remote /dev/ttyUSB0\"]",
+  "esp_idf.gdbinitFile.description": "Путь к файлу gdbinit для адаптера отладки ESP-IDF",
+  "esp_idf.debuggers.text.description": "Команда для выполнения"
+}
```

Comment:
There are some missing translations in the file. English translation in current master has 80 lines, while only 76 are here